src/slack/actions/index.ts

```diff
@@ -0,0 +1,8 @@
+import { App } from '@slack/bolt';
+import logger from '../../logger';
+import registerTeam from './registerTeam';
+
+export default function register(app: App): void {
+  logger.info('Registering action listeners');
+  registerTeam(app);
```

Comment:
I'm good with this for now, but in the future, I want to add an auto-import for all files in the folder